src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3727,13 +3636,6 @@ bool AdaptDispatch::EndHyperlink()
 // - True if handled successfully. False otherwise.
 bool AdaptDispatch::DoConEmuAction(const std::wstring_view string)
 {
-    // Return false to forward the operation to the hosting terminal,
```

Comment:
this was another instance (like clipboard) of "conhost doesn't *need* to do this, because the connected terminal will do it". Same Q as clipboard

Reply:
If this code is expensive, we should make it cheap, because this would benefit WT as well.